packages/scripts/test/optimistic-oracle-umip/3_Verify.ts

```diff
@@ -0,0 +1,61 @@
+// This script verify that the upgrade was executed correctly. It can be
+// run on a local hardhat node fork of the mainnet or can be run directly on the mainnet to execute the upgrade
+// transactions. To run this on the localhost first fork mainnet into a local hardhat node by running:
+// HARDHAT_CHAIN_ID=1 yarn hardhat node --fork https://mainnet.infura.io/v3/<YOUR-INFURA-KEY> --port 9545 --no-deploy
+// Then execute the script from core:
+// OPTIMISTC_ORACLE_V2=<OPTIMISTC_ORACLE_V2_ADDRESS> yarn workspace @uma/scripts hardhat run ./test/optimistic-oracle-umip/3_Verify.ts --network localhost
+
+const hre = require("hardhat");
+
+const assert = require("assert").strict;
+
+const { RegistryRolesEnum } = require("@uma/common");
+
+import { Governor, Finder, Registry } from "@uma/core/contract-types/ethers";
+
+const { getAddress } = require("@uma/contracts-node");
+
+const OPTIMISTIC_ORACLE_V2 = "OptimisticOracleV2"; // TODO use interfaceName.OptimisticOracle
+const deployed_optimistic_oracle_address = process.env["OPTIMISTC_ORACLE_V2"];
+
+const getContractInstance = async <T>(contractName: string): Promise<T> => {
```

Comment:
Since this is in two places, it could make sense to move it to some common location, like a utils file. Not strongly opinionated, but it does seem generally useful!